nsls2tools/csx1/plans/suspenders.py

```diff
@@ -0,0 +1,36 @@
+from bluesky.suspenders import (SuspendBoolHigh,
+                                SuspendBoolLow,
+                                SuspendFloor,
+                                SuspendCeil,
+                                SuspendInBand,
+                                SuspendOutBand)
+
+from ophyd import EpicsSignal
+
+### Enable this line for standard GU operation
```

Comment:
Intentionally disabled? Perhaps this is awaiting the improved suspenders I've been talking about adding to bluesky, which you allow suspenders to block certain operations (triggering the FCCD while the beam is down) but allow others (moving a motor while the beam is down).

Reply:
Someone commented out for using the beamline during the shutdown.  Then all of it was commented out to prevent people from thinking that the suspenders were enabled.  Perhaps this should set up and define the suspender, but not install it.  We could print to the screen how to install the suspender.  THen the user can put that line in the local startup file that includes user specific `md`.   @stuwilkins @cmazzoli @wenhu ?  
Then we can re-evaluate things once the improved suspender capability is launched.